Lib/asyncio/proactor_events.py

```diff
@@ -426,6 +427,120 @@ def _pipe_closed(self, fut):
             self.close()
 
 
+class _ProactorDatagramTransport(_ProactorBasePipeTransport):
```

Comment:
Note: can_write_eof() is not implemented, but _SelectorDatagramTransport does not implement it neither, so it's fine.